docs/site/src/demos/dialogs/FullScreenDialog.js

```diff
@@ -0,0 +1,77 @@
+// @flow weak
+
+import React, { Component } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import Button from 'material-ui/Button';
+import { Dialog } from 'material-ui/Dialog';
+import {
+  List,
+  ListItem,
+  ListItemText,
+} from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+import AppBar from 'material-ui/AppBar';
+import Toolbar from 'material-ui/Toolbar';
+import IconButton from 'material-ui/IconButton';
+import Text from 'material-ui/Text';
+import CancelIcon from 'material-ui/svg-icons/cancel';
+import Slide from 'material-ui/transitions/Slide';
+
+const styleSheet = createStyleSheet('FullScreenDialog', () => ({
+  appBar: {
+    position: 'relative',
+    width: '100%',
+  },
+  flex: {
+    flex: 1,
+  },
+}));
+
+export default class FullScreenDialog extends Component {
+  state = {
+    open: false,
+  };
+
+  handleRequestClose = () => this.setState({ open: false });
+
+  render() {
+    const classes = this.context.styleManager.render(styleSheet);
+    return (
+      <div>
+        <Button onClick={() => this.setState({ open: true })}>
+          Open full-screen dialog
+        </Button>
+        <Dialog
+          fullscreen
+          open={this.state.open}
+          onRequestClose={this.handleRequestClose}
+          transition={<Slide direction="up" />}
+        >
+          <AppBar className={classes.appBar}>
+            <Toolbar>
+              <IconButton contrast onClick={this.handleRequestClose}>
+                <CancelIcon />
```

Comment:
That icon don't feel right
![capture d ecran 2017-01-23 a 21 03 29](https://cloud.githubusercontent.com/assets/3165635/22220560/7191718a-e1af-11e6-8eac-90ca0fd3009c.png)